ansible/files/steps/zeppelin/zeppelin.sh

```diff
@@ -17,6 +18,17 @@ nohup Xvfb &
 
 echo 'export DISPLAY=:0.0' | sudo tee -a /etc/zeppelin/conf/zeppelin-env.sh
 
+# Preload Scala packages
+repositories="https://oss.sonatype.org/content/repositories/snapshots"
```

Comment:
Yes, this is a trusted source and many important Scala projects are hosted there, like [breeze](https://github.com/scalanlp/breeze). It would be great to have local mirrors for this and other Maven repositories and possibly have our own repository as publishing a Scala package isn't very straightforward.